src/DynamicRupture/Output/ReceiverBasedOutput.h

```diff
@@ -54,8 +54,11 @@ class ReceiverOutput {
     size_t ltsId{};
     int nearestGpIndex{};
     int nearestInternalGpIndex{};
+    int gpIndexFused{};
```

Comment:
It would be good to rename that one to show that it's also relevant for non-fused simulations (and in fact, the main index now).
Maybe just `index`/`dataIndex`?

Reply:
Ok; sounds good to me.